### PR TITLE
Globals: Various fixes

### PR DESCRIPTION
**Pull request by udidol**

### CODE REVIEW

File: assets/dev/scss/frontend/widgets/heading.scss [hunk 1]
```diff
@@ -2,7 +2,8 @@
 // Heading
 //
 
-.elementor-heading-title {
+// The double selector here is for specificity reasons - to avoid being overridden by default Globals CSS.
```
Comment (kobizz) [inline assets/dev/scss/frontend/widgets/heading.scss:5]: Check what happens in theme style typography

**Review by kobizz — CHANGES_REQUESTED** [review]

File: assets/dev/js/editor/controls/color.js [hunk 2]
```diff
@@ -184,6 +184,8 @@ export default class extends ControlBaseDataView {
 		// Empty the value saved in the control.
 		this.setValue( '' );
 
+		this.globalValue = null;
```
[the remark this answers is not in the document]
Comment (kobizz) [reply assets/dev/js/editor/controls/color.js:187]: @udidol What about this?

**Review by kobizz — CHANGES_REQUESTED** [review]

File: assets/dev/js/editor/controls/color.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (udidol) [reply assets/dev/js/editor/controls/color.js:187]: I am in favor of leaving it. The error which caused me to add it was not actually caused by the absence of it. But - when I added it, the clear action worked as it should even despite the error. What do you think?

File: core/kits/manager.php [hunk 3]
```diff
@@ -119,7 +119,7 @@ public function localize_settings( $settings ) {
 				'global_colors' => __( 'Global Colors', 'elementor' ),
 				'invalid' => __( 'Invalid', 'elementor' ),
 				'color_cannot_be_deleted' => __( 'System Colors can\'t be deleted', 'elementor' ),
-				'typography_cannot_be_deleted' => __( 'System Typography can\'t be deleted', 'elementor' ),
+				'font_cannot_be_deleted' => __( 'System Font can\'t be deleted', 'elementor' ),
```
Comment (kobizz) [inline core/kits/manager.php:122]: Where is this key implemented?

**Review by kobizz — CHANGES_REQUESTED** [review]

File: core/kits/manager.php (hunk 3 above)
Comment (udidol) [reply core/kits/manager.php:122]: kit repeater-row.js

**Review by kobizz — APPROVED** [review]

**Review by matipojo — APPROVED** [review]

**Pull request merged**